### ISSUE TITLE
Crash on playback of any video

### ISSUE THREAD

**Issue by kejml** [ytmdesktop/ytmdesktop#1324]
### Consent

- [X] I verified that there is no open/closed issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

Whenever I try to click play any video on my Linux (Ubuntu) machine, the application crashes immediately.

```
Environment Details:
  YouTube Music Desktop App 2.0.0
  Electron 28.2.0
  Linux 5.15.0-92-generic

Name: Error
Message: connect ECONNREFUSED 172.217.23.118:443
Cause: Unknown

Error: connect ECONNREFUSED 172.217.23.118:443
  at TCPConnectWrap.afterConnect [as incomplete] (node:net:1555:16)
```

I'm sitting behind a corporate proxy, but it doesn't block YouTube itself in a regular browser. Also previous 1.X version of ytmdesktop worked for me.

PS: The `Copy to Clipboard and Exit` doesn't copy anything to the clipboard at all for me. I had to type the error message manually :( 

### Expected Behavior

Video should start playing normally

### Steps To Reproduce

1. Open the application
2. Click on any video to play it
3. Application shows error popup

### YTMDesktop

2.0.0

### OS

Ubuntu

### OS Version

20.04

### Arch

x64

### Installation way

.deb

**Comment by kejml** [comment]
BTW I'm running the version from https://github.com/ytmdesktop/ytmdesktop/actions/runs/7688358807 , to overcome #1247

**Comment by Alipoodle** [comment]
Do you have Notifications on for each track playing?
the IP shared is a Google LLC hosted IP~ so this sounds like something of Google being blocked.
I also run on a Corp network and have a few things blocked like Google's Analytic pages but I don't crash from it 😅

Worth seeing what Domains are blocked when you run YTM in a Web browser when you play a track.

**Comment by kejml** [comment]
Yes, I had notifications enabled, without that, it works, at least for a few minutes. Thanks for the hint!

Regarding the blocked domains, some Google services are blocked here (e.g., Google Meet), but I don't see any blocked domains while opening YouTube Music in the browser in Firefox console. I can see only some `401`s while it tries to send a POST request to `https://play.google.com/log?format=json&hasfast=true&authuser=0`. But AFAIK a corporate proxy shouldn't cause that. I can access play.google.com from a browser without any obvious issues.

**Comment by Alipoodle** [comment]
Can I check that you tried to play a song and that you only got a 401 from the `play.google.com/log` ?
![image](https://github.com/ytmdesktop/ytmdesktop/assets/17199186/1ee38c26-e992-4986-8d85-81f85899713f)

We call to the `https://lh3.googleusercontent.com/` which some networks have blocked previously for me even randomly.
But that's the URL we would call to get the Thumbnail for the Notification on YTMD for the notification.

**Comment by kejml** [comment]
Yep, here is a screenshot while playing a song in Firefox:

![image](https://github.com/ytmdesktop/ytmdesktop/assets/4223239/99a94d6a-bb09-4756-82da-a20657d2f092)

If there is a way how to get the logs from YTMDesktop, I can definitely get you those

`https://lh3.googleusercontent.com/` doesn't seem to be blocked in our network, I get a Google's `400` when opening it in the browser
